src/Spectre.libGenetic/MutationOperator.cpp

```diff
@@ -52,6 +61,24 @@ Individual MutationOperator::operator()(Individual &&individual)
         };
         std::transform(individual.begin(), individual.end(), individual.begin(), swap_random_bits);
     }
-    return individual;
+
+    // @gmrukwa: TODO: test this behaviour
```

Comment:
Remove `TODO`, since it seems to be tested already.

Reply:
done